test/qunit/model/FHIRModel.unit.js

```diff
@@ -666,6 +666,23 @@ sap.ui.define([
 		assert.deepEqual(mGoalParameters.birthdate, mParameters.urlParameters.birthdate);
 	});
 
+	QUnit.test("filtering and sorting a list with a single object", function (assert) {
+		this.loadDataIntoModel("Patients");
+		var oGenderFilter = new sap.ui.model.Filter({ path: "gender", operator: FHIRFilterOperator.EQ, value1: "male" });
+		var oSort = new sap.ui.model.Sorter("birthdate", true);
+
+		this.oListBinding.sort(oSort);
+		this.oListBinding.filter(oGenderFilter);
```

Comment:
Let's add an additional test case when sort and filter is called without any value defined.

Reply:
done

Reply:
@flovogt Done